package.json

```diff
@@ -40,6 +41,26 @@
         "@vue/compiler-sfc": "^3.0.5",
         "core-js": "^3.15.2",
         "sass": "^1.35.1",
-        "vite": "^2.3.7"
+        "vite": "^2.3.7",
+        "auto-changelog": "2.3.0",
+        "release-it": "14.10.0"
```

Comment:
Please move it to devDependencies. We don't want to have unused packaged in PROD Docker image (https://github.com/louislam/uptime-kuma/pull/122)

Reply:
The two dependencies you mentioned are already devDependencies :)

Reply:
Sorry, I didn't see full file before :P